dev/TreeView/TreeViewList.cpp

```diff
@@ -320,6 +320,15 @@ void TreeViewList::PrepareContainerForItemOverride(winrt::DependencyObject const
     {
         bool hasChildren = itemContainer.HasUnrealizedChildren() || itemNode->HasChildren();
         itemContainer.GlyphOpacity(hasChildren ? 1.0 : 0.0);
+        if (itemContainer.IsExpanded() != itemNode->IsExpanded()) {
+            auto dispatcher = winrt::Window::Current().Dispatcher();
```

Comment:
>auto dispatcher = winrt::Window::Current().Dispatcher(); [](start = 12, length = 56)

I think this is supposed to come from dispatcher pool, rather than window.current to be islands safe.  @kmahone might be able to confirm?

Reply:
Yeah. Use DispatcherHelper (dev\inc\DispatcherHelper.h) to get the correct behavior.